content/en-us/reference/engine/classes/AssetService.yaml

```
@@ -133,8 +135,10 @@ methods:
     summary: |
       Creates a new version for an existing asset from the given object.
```

Use Upload here too for consistency?
```suggestion
      Uploads a new version for an existing asset from the given object.
```